libafl/src/events/mod.rs

```diff
@@ -621,6 +624,10 @@ impl<S> EventFirer for NopEventManager<S>
 where
     S: State,
 {
+    fn sample(&self, _corpus_count: usize) -> bool {
```

Comment:
As noted earlier, let's do a `throttle` function instead (or even call it `should_skip`/`should_send` to be flexible)

Reply:
DONE

Reply:
Awsome